lib/beaker/host_prebuilt_steps.rb

```diff
@@ -367,13 +367,15 @@ def copy_ssh_to_root host, opts
     def hack_etc_hosts hosts, opts
       etc_hosts = "127.0.0.1\tlocalhost localhost.localdomain\n"
       hosts.each do |host|
-        ip = host['vm_ip'] || host['ip'].to_s
-        hostname = host[:vmhostname] || host.name
-        domain = get_domain_name(host)
-        etc_hosts += "#{ip}\t#{hostname}.#{domain} #{hostname}\n"
+        unless host['platform'] =~ /windows/
```

Comment:
@petems could provide more insight but looking at the logic the get_domain_name and set_etc_hosts functions both are coded with *nix / cygwin paths, I would agree that that omitting host file hacking works just fine for single node test cases, but it does in fact break multi node test cases across the board. 

I will get the appropriate logic in place.